app/policies/comment_policy.rb

```diff
@@ -41,7 +41,7 @@ def moderate?
 
   def moderator_create?
     # NOTE: Here, when we say "moderator", we mean "tag_moderator"
-    user_moderator? || user_any_admin?
+    user_trusted? || user_moderator? || user_any_admin?
```

Comment:
Maybe this is what PJ meant [here](https://github.com/forem/forem/pull/20158#discussion_r1335862671) We could re-use the `accesses_mod_response_templates?` method (and rename it if need be). 

```suggestion
Authorizer.for(user: user).accesses_mod_response_templates?
```